Homework6/src/CaesarCodec.java

```diff
@@ -0,0 +1,59 @@
+import java.util.HashMap;
+
+import java.util.Map;
+//*Я не списывал у Ярика)
+
+public class CaesarCodec  {
+            Map mapsm,mapsd;
```

Comment:
нужно типизировать переменные с дженериками
```suggestion
            Map<Character, Character> mapsm,mapsd;
```